src/posts.fr-CA.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: layouts/posts.njk
+title: Articles
+lang: fr-CA
+translationKey: posts
+eleventyNavigation:
+  parent: fr-CA
+  key: Articles
+  order: 3
+permalink: "/{{ config.languages[lang].slug }}/articles/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber + 1 }}/{% endif %}"
```

Comment:
I assume this issue is related to this line, but it might be something else. If you set the site to default to `fr-CA`. The URLs for Home and About have `/en/` in the path for the English pages, but Posts does not. Conversely, the French pages for Home and About do not have `/fr/` in the URL but the Posts page does. 

Navigating from the Posts page **does not** break the language selection.

Reply:
Fixed in 2150a10 so that changing defaultLanguage updates these permalinks appropriately.